src/library-authoring/containers/UnitInfo.tsx

```diff
@@ -14,6 +20,47 @@ import {
 } from '../common/context/SidebarContext';
 import ContainerOrganize from './ContainerOrganize';
 import messages from './messages';
+import componentMessages from '../components/messages';
+import ContainerDeleter from '../components/ContainerDeleter';
+import { useContainer } from '../data/apiHooks';
+
+type ContainerMenuProps = {
+  containerId: string,
+  displayName: string,
+};
+
+const UnitMenu = ({ containerId, displayName }: ContainerMenuProps) => {
+  const intl = useIntl();
+
+  const [isConfirmingDelete, confirmDelete, cancelDelete] = useToggle(false);
+
+  return (
+    <>
+      <Dropdown id="unit-info-dropdown">
+        <Dropdown.Toggle
+          id="unit-info-menu-toggle"
+          as={IconButton}
+          src={MoreVert}
+          iconAs={Icon}
+          variant="primary"
+          alt={intl.formatMessage(componentMessages.collectionCardMenuAlt)}
```

Comment:
nit: we are using an incorrect string here
```suggestion
          alt={intl.formatMessage(componentMessages.containerCardMenuAlt)}
```